ext/sockets/sockets.stub.php

```diff
@@ -2156,3 +2161,19 @@ function socket_wsaprotocol_info_import(string $info_id): Socket|false {}
 
 function socket_wsaprotocol_info_release(string $info_id): bool {}
 #endif
+
+#ifdef AF_PACKET
+final class SocketEthernetInfo
+{
+    /** @readonly **/
```

Comment:
Why did you use the `@readonly` annotation  instead of real readonly properties?

Reply:
This object will be rewritten in a following PR anyway.